provision/common/Taskfile.yaml

```diff
@@ -8,7 +8,6 @@ vars:
   KC_DB_POOL_INITIAL_SIZE: '{{default "5" .KC_DB_POOL_INITIAL_SIZE}}'
   KC_DB_POOL_MAX_SIZE: '{{default "10" .KC_DB_POOL_MAX_SIZE}}'
   KC_DB_POOL_MIN_SIZE: '{{default "5" .KC_DB_POOL_MIN_SIZE}}'
-  KC_STORAGE: '{{default "" .KC_STORAGE}}'
```

Comment:
so we are effectively removing the `KC_STORAGE` as an `.env` option ? I think that makes sense.

Reply:
Yes, it is being removed.